### PR TITLE
Updated changelog as part of the release

### PR DESCRIPTION
**Pull request by commjoen**
Thank you for submitting a Pull Request to the Mobile Application Security Verification Standard. Please make sure that:

- [ ] Your PR is linked to an issue.
- [ ] Your PR follows the [contribution guidelines](https://github.com/OWASP/owasp-masvs/blob/master/CONTRIBUTING.md "Contribution guidelines")
- [ ] All links are valid and use HTTPS and have been formatted as [TEXT](URL “NAME”)
- [ ] In case of a translation: have you made sure that the translation from EN is as close to the EN document as possible?

This is a release-notes PR.

If your pull-request has a focus of rewriting a requirement and/or adding a new one, please make sure you have a backing issue in which your change has been discussed and agreed. If there is none, then your PR will be closed without further review.

### CODE REVIEW

**Comment by OMTGreleaser** [comment]
Results for commit eabb18fc0cf9ebc7c8af7bd0bbb69376db2d0818: Broken link result: 0 .
  
 Markdown result errorlength: 0 .

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,21 @@
 
 This document is automatically generated at {{gitbook.time}}
 
+## V1.1.4 4 July 2019 - Summit edition (WIP)
+
+The following changes are part of release 1.1.4:
+
+- Fix all markdown issues.
+- Updates in the French and Spanish translations.
+- Translated the changelog in Chinese (ZHTW) and Japanese.
```
Comment (cpholguera) [inline CHANGELOG.md]: **to** Chinese?

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,21 @@
...
+- Automated verification of the the markdown and links.
```
Comment (cpholguera) [inline CHANGELOG.md]: Automated verification of markdown and links. ?

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,21 @@
...
+- Added identification codes to the requirements, which will be included in the future version of the MSTG in order to find the recommendations and testcases easily.
```
Comment (cpholguera) [inline CHANGELOG.md]: testcases -> test cases

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,21 @@
...
+- Reduced the repo size and added Generated to the .gitignore.
+- Added a Code of Conduct & Contributing guidelines.
+- Added a Pull-Request template.
+- Updated the sync with the repo in use for hosting the Gitbook website.
```
Comment (cpholguera) [inline CHANGELOG.md]: Updated the repo sync for ...?

**Pull request merged**